pkgs/applications/editors/jetbrains/default.nix

```
@@ -347,20 +341,12 @@ rec {
         postInstall =
           (attrs.postInstall or "")
           + lib.optionalString (stdenv.hostPlatform.isLinux) ''
-            (
-              cd $out/rider
-
-              ls -d $PWD/plugins/cidr-debugger-plugin/bin/lldb/linux/*/lib/python3.8/lib-dynload/* |
-              xargs patchelf \
-                --replace-needed libssl.so.10 libssl.so \
-                --replace-needed libcrypto.so.10 libcrypto.so \
-                --replace-needed libcrypt.so.1 libcrypt.so
-
-              for dir in lib/ReSharperHost/linux-*; do
-                rm -rf $dir/dotnet
-                ln -s ${dotnet-sdk}/share/dotnet $dir/dotnet
-              done
-            )
+            ${patchSharedLibs}
+
+            for dir in $out/rider/ReSharperHost/linux-*; do
```

This path was `$out/rider/lib/ReSharperHost/linux-*` before. Is the change intentional?

Could be related to https://github.com/NixOS/nixpkgs/pull/419026#issuecomment-3034879811

```suggestion
            for dir in $out/rider/lib/ReSharperHost/linux-*; do
```

---

Thanks, that was indeed not intended.

---

That solved it, just tested the most recent commits!